native/audio-host/win/verify.ps1

```
@@ -64,6 +66,66 @@ foreach ($case in @(@{n = 'NOISY'; p = $noisy.Id }, @{n = 'QUIET'; p = $quiet.Id
+
+$rootPid = (Get-CimInstance Win32_Process -Filter "ParentProcessId=$($launcher.Id) AND Name='powershell.exe'" |
+    Select-Object -First 1).ProcessId
+$childPid = $null
+if ($rootPid) {
+    $childPid = (Get-CimInstance Win32_Process -Filter "ParentProcessId=$rootPid AND Name='powershell.exe'" |
+        Select-Object -First 1).ProcessId
+}
+Write-Output ("TREE: root(parent)={0} audio-child={1}" -f $rootPid, $childPid)
+
+$listedIds = @()
+try { $listedIds = @((& $exe --list | ConvertFrom-Json) | ForEach-Object { $_.id }) } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)